cmake/CorrosionGenerator.cmake

```diff
@@ -111,6 +119,7 @@ function(_generator_add_package_targets)
             _corrosion_add_library_target(
                 WORKSPACE_MANIFEST_PATH "${workspace_manifest_path}"
                 TARGET_NAME "${target_name}"
```

Comment:
I'm wondering: Is the original `target_name` even still used after we defined `target_name_cmake`? Is there a reason why we would need both target_name and target_name_cmake?

Reply:
Yes, it is used to form the lib file names.